typescript/infra/config/environments/mainnet3/warp/configGetters/getRenzoEZETHWarpConfig.ts

```diff
@@ -222,6 +298,8 @@ export const getRenzoEZETHWarpConfig = async (): Promise<
     await Promise.all(
       chainsToDeploy.map(
         async (chain): Promise<[string, HypTokenRouterConfig]> => {
+          const { mailbox, merkleTreeHook, interchainGasPaymaster } =
+            (await registry.getChainAddresses(chain))!;
```

Comment:
should we maybe do this to be extra safe
```
const addresses = await registry.getChainAddresses(chain);
if (!addresses) {
  throw Error(`No addresses for chain ${chain}`);
}
const { mailbox, merkleTreeHook, interchainGasPaymaster } = addresses;
```

Reply:
Added `assert()`